site_scons/prereq_tools/base.py

```diff
@@ -332,11 +332,14 @@ def checkout_commit(self, subdir):
     def apply_patches(self, subdir, patches):
         """ git-apply a certain hash """
         if patches is not None:
-            for patch in patches:
+            for patch in patches.keys():
                 print("Applying patch %s" % (patch))
-                commands = [['git', 'apply', patch]]
-                if not RUNNER.run_commands(commands, subdir=subdir):
-                    raise DownloadFailure(self.url, subdir)
```

Comment:
why not do this anymore?

Reply:
I was playing around with code and removed it, will restore.